build/dependencies.props

```diff
@@ -4,6 +4,7 @@
     <CoreFxVersion>4.3.0</CoreFxVersion>
     <InternalAspNetCoreSdkVersion>2.1.0-*</InternalAspNetCoreSdkVersion>
     <NETStandardImplicitPackageVersion>$(BundledNETStandardPackageVersion)</NETStandardImplicitPackageVersion>
+    <NETStandardLibraryNETFrameworkVersion>2.0.0-*</NETStandardLibraryNETFrameworkVersion>
```

Comment:
Doesn't seem like the two packages are updated in lock step:

* https://dotnet.myget.org/feed/dotnet-core/package/nuget/NETStandard.Library.NETFramework
* https://dotnet.myget.org/feed/dotnet-core/package/nuget/NETStandard.Library

Think this is fine as is.